### ISSUE TITLE
Missing commits?

### ISSUE THREAD

**Issue by icemac** [zopefoundation/initgroups#4]
There is already a 3.0 release on PyPI: https://pypi.python.org/pypi/initgroups/3.0 whereas here on GitHub the change log has 3.0.0 as unreleased: https://github.com/zopefoundation/initgroups/blob/master/CHANGES.rst

PyPI claims that @hannosch did this release at 2016-07-19, 12:04.

@hannosch do you still have the missing change sets at hand?

**Comment by hannosch** [comment]
I didn't have the repo checked out locally anymore. Looks like I must have forgotten to push the changes :(

I've created a 3.x branch, applied the changes from the PyPi release onto the state of the master branch before the release date and created a 3.0 tag from this. Finally added the missing release to the changelog on master and bumped the version there.

I did a local release from the 3.0 tag and diffed it against the zipfile on PyPi and they match now.
